src/resources/css/Common.module.css

```diff
@@ -2,6 +2,12 @@ body {
   margin: 0;
 }
 
+html {
+  scrollbar-gutter: stable;
```

Comment:
Also, when we open a dialog, the backdrop doesn't fill the entire screen. It looks a bit weird on small screens especially. It might be a better solution to add a permanent scrollbar, or add a condition to this so that this styling only applies to larger screens. 
```suggestion
 overflow-y: scroll;
```
<img width="545" alt="Screenshot 2025-04-11 at 11 51 46" src="https://github.com/user-attachments/assets/3d134e7b-272c-4401-8a54-93a5f7674933" />

Reply:
Yeah, that's the problem I was talking about in the PR description, that I haven't been able to find a solution for 😅
The permanent scrollbar is an interesting thought. However, in my experience, the scrollbar automatically disappears when the dialog opens, leaving only the gutter behind. So I'm not sure if always having a scrollbar there will make a difference....

Oh, but, it doesn't show up on iPad and mobile view for me, though (when in simulated touch screens). Is that just me? 🤔

![image](https://github.com/user-attachments/assets/ecfb5605-0e6d-4f88-a945-8a77d557b5aa)

Reply:
Yes. sorry. Its probably correct that it dosn't show up on mobile screens... so this might wont be an issue. I was testing with firefox on descktop.